src/hiero_sdk_python/client/client.py

```
@@ -47,6 +50,91 @@ def __init__(self, network: Network = None) -> None:
+        
+        if network:
+            network_name = network
+        else:
+            network_name = os.getenv('HEDERA_NETWORK') or os.getenv('NETWORK') or 'testnet'
+
+        network_name = network_name.lower()
+        
+        try:
+            client = cls(Network(network_name))
+        except ValueError:
+            raise ValueError(f"Invalid network name: {network_name}")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)